api/ccapi/include/tensor_dim.h

```diff
@@ -49,12 +49,14 @@ class TensorDim {
 
   /**
    * @brief Tensor Data Type.
-   * Currently support QINT4, QINT8, UINT16, FP16 & FP32
+   * Currently support QINT4, QINT8, UINT8, UINT16, UINT32, FP16 & FP32
    */
   enum class DataType {
     QINT4,  /** quantized int 4*/
     QINT8,  /** quantized int 8*/
+    UINT8,  /** unsigned int 8 bit */
     UINT16, /** unsigned int 16 bit */
+    UINT32, /** unsigned int 32 bit */
```

Comment:
please check tensor_dim.cpp to reflect this patch. (e.g., `getDataTypeSize()`, `operator<<`)

Reply:
I updated the  [`tensor_dim.cpp`](https://github.com/nnstreamer/nntrainer/pull/2735/files#diff-8a77082124d610f5fb8ab7e7cc4efab8ca088a9f98ecf8c5288cfbecd55d91f8). Thank you for the comment.